### DISCUSSION TITLE
Keyboard Input

### DISCUSSION THREAD

**Discussion by UkrainianBanderasCat** [UkrainianBanderasCat/Digital-Logic-Sim2#32] [General]
I added keyboard input, but it doesn't read Alt, Ctrl, ShiftLeft, and ShiftRight, F1-F12, Tab, Arrows, or Capslock and has some problems with symbols. It reads lowercase letters, uppercase letters, numbers, backspace, enters, and space. Can someone help me to fix that?

**Comment by AlphaCodingPilot** [comment]
What does the keyboard chip even do?

**Reply by UkrainianBanderasCat** [reply]
When you press a key on the keyboard the chip converting this key to ASCII and than to Binary and it send binary code of pressed key
